data/json/recipes/chem/fuel.json

```diff
@@ -45,6 +45,7 @@
   },
   {
     "result": "biodiesel",
+    "byproducts": [ [ "chem_glycerol" ] ],
```

Comment:
Did you check that the quantity of glycerol produced here is appropriate to the proportions in the recipe?